azurerm/internal/services/powerbi/registration.go

```diff
@@ -26,6 +26,6 @@ func (r Registration) SupportedDataSources() map[string]*schema.Resource {
 // SupportedResources returns the supported Resources supported by this Service
 func (r Registration) SupportedResources() map[string]*schema.Resource {
 	return map[string]*schema.Resource{
-		"azurerm_powerbi_embedded": resourceArmPowerBIEmbedded(),
+		"azurerm_powerbi_embedded": resourcePowerBIEmbedded(), // todo should this be renamed to `azurerm_powerbi_embedded_capacity` ?
```

Comment:
I don't think that's necessary here, so this can be removed

```suggestion
		"azurerm_powerbi_embedded": resourcePowerBIEmbedded(),
```